urllib3/exceptions.py

```diff
@@ -193,6 +196,25 @@ class ResponseNotChunked(ProtocolError, ValueError):
     pass
 
 
+class IncompleteRead(httplib_IncompleteRead):
```

Comment:
Hrmmmmmm.

Sorry @nateprewitt, I know I should have dealt with this earlier, but I think that this should probably dual-inherit from both IncompleteRead and `HTTPError`, so that people indiscriminately catching all urllib3 errors catch this too.